addons/web_editor/static/src/xml/snippets.xml

```diff
@@ -78,6 +78,7 @@
     </t>
     <t t-name="web_editor.color.combination.preview">
         <div class="o_we_cc_preview_wrapper d-flex justify-content-between">
+            <span class='fa fa-check o_we_color_combination_icon text-success d-none'/>
```

Comment:
- `"` instead of `'` in XML please
- Maybe you don't need that XML, can't it be like the other check icons in the editor? Pure scss with a ::before element?
- Move this commit in the 14.0 PR too, could be considered a bug fix.